Kalenteri/Backend/Services/OrderService.cs

```diff
@@ -1,7 +1,48 @@
+using Kalenteri.Backend.Models;
+using Microsoft.Azure.Cosmos;
+
 namespace Kalenteri.Backend.Services;
 
 public class OrderServices
 {
-    // Business logic here
-}
+    private const string EndpointUrl = "https://kalentericosmosdbtest.documents.azure.com:443/";
+    private const string PrimaryKey = "ON6nEfIrVtiQduJfdh9Bc0Xp6dPHCnAWdmmsvnnxvNRKj2PzH7IaPmGnQrwSJBrmBntNJCz5M8ymACDbeQsQ2w==";
+    private const string DatabaseId = "kalenteri";
+    private const string ContainerId = "cont1";
+
+    private static Order _order = null; 
```

Comment:
Tän pitäis olla nullable. Ainakin meikäläisen language server heittää warningia muuten.

```suggestion
    private static Order? _order = null; 
```